test/lib/Elastica/Test/ClientTest.php

```diff
@@ -26,6 +26,16 @@ public function testConstruct()
     /**
      * @group functional
      */
+    public function testGetVersion()
+    {
+        $client = $this->_getClient();
+        $this->assertNotEmpty($version = $client->getVersion());
+        $this->assertSame($version, $client->getVersion());
```

Comment:
I think it is fine to "hardcode" the minimal compatible version. `>=2.0.0` will work for all future versions. I agree if we "hardcode" it, it should be in a variable like `MINIMAL_COMPATIBLE_VERSION` for reusability and this makes it easier to update it.